dashboard/client/src/api.ts

```diff
@@ -294,19 +294,18 @@ export const getErrors = (nodeIp: string, pid: number | null) =>
     pid: pid ?? "",
   });
 
-export type LogsResponse = {
-  logs: LogsByPid;
-};
-
-export type LogsByPid = {
-  [pid: string]: string[];
-};
-
-export const getLogs = (nodeIp: string, pid: number | null) =>
-  get<LogsResponse>("/node_logs", {
-    ip: nodeIp,
-    pid: pid ?? "",
+export const getLogs = async (nodeIp: string, pid: number) => {
```

Comment:
get logs is a streaming API. Can you verify if this works with large # of logs?

Reply:
I think this will work not worse than how the dashboard works today. a streaming log UI is out of scope for this fix.